vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -155,10 +156,46 @@ void FlyoutShadowNode::createView() {
   m_flyoutClosedRevoker =
       m_flyout.Closed(winrt::auto_revoke, [=](auto &&, auto &&) {
         auto instance = wkinstance.lock();
-        if (!m_updating && instance != nullptr)
+        if (!m_updating && instance != nullptr) {
+          // When the flyout closes, attempt to move focus to
+          // its anchor element to prevent cases where focus can land on
+          // an outer flyout content and therefore trigger a unexpected flyout
+          // dismissal
+          winrt::FocusManager::TryFocusAsync(
+              m_targetElement, winrt::FocusState::Programmatic);
```

Comment:
Will all Flyout have a valid targetElement? It's possible that a popup without target

Reply:
It looks like m_targetElement will be set to something. If it's null at the time of updating properties, SetTargetFrameworkElement sets m_targetElement to winrt::Window::Current().Content().

I can add a null check and just not do something here if it's null though to be safe.